File: httpsnippet/src/test/resources/output/node/unirest/query.js
var unirest = require("unirest");

var req = unirest("GET", "http://mockbin.com/har?key=value");

req.query({"foo":["bar","baz"],"baz":["abc"]});


req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: src/test/resources/output/go/native/query.go
package main

import (
	"fmt"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "http://mockbin.com/har?baz=abc&foo=bar&foo=baz&key=value"

	req, _ := http.NewRequest("GET", url, nil)

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}
